pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -241,8 +242,13 @@ func (s *Server) initInprocessAnalysisController(args *PilotArgs) error {
 	processingArgs := settings.DefaultArgs()
 	processingArgs.KubeConfig = args.RegistryOptions.KubeConfig
 	processingArgs.WatchedNamespaces = args.RegistryOptions.KubeOptions.WatchedNamespaces
-	processingArgs.MeshConfigFile = args.MeshConfigFile
 	processingArgs.EnableConfigAnalysis = true
+	meshSource := mesh.NewInmemoryMeshCfg()
```

Comment:
Does this mean the InprocessAnalysisController will always use the defaultMeshconfig? How about the cases that user defines the meshConfig via file or ConfigMap?

Reply:
No, L247-250 update it when it changes

Reply:
ah, got it